osi_vehicle.proto

```diff
@@ -0,0 +1,829 @@
+        //
+        // Unit: [N*m/deg]
+        //
+        optional double steering_springstiffness = 2;
+
+        // Damping of the steering in Nm*s/deg.
+        //
+        // Unit: [N*m*s/deg]
+        //
+        optional double steering_damping = 3;
+
+        // Friction of the steering in Nm.
+        //
+        // Unit: [N*m]
+        //
+        optional double steering_friction = 4;
+
+        // Describes the state of the passenger's hands related to the steering
+        // wheel (mostly driver).
+        //
+        optional Occupant.Classification.SteeringControl steering_control = 5;
+    }
+
+    //
+    // \brief The focus here is on the wheels.
+    // It is made usage of the wheel-message to shorten the code.
+    //
+    message VehicleWheels
+    {
+        // Contains the physical description of the front-left wheel.
+        //
+        optional Wheel wheel_front_left = 1;
+
+        // Contains the physical description of the front-right wheel.
+        //
+        optional Wheel wheel_front_right = 2;
+
+        // Contains the physical description of the rear-left wheel.
+        //
+        optional Wheel wheel_rear_left = 3;
+
+        // Contains the physical description of the rear-right wheel.
+        //
+        optional Wheel wheel_rear_right = 4;
+    }
+
+    //
+    // \brief The focus here is on the physical description of a wheel.
+    //
+    message Wheel
+    {
+        // Dry friction is a force that opposes the relative lateral motion of two solid surfaces in contact. 
+        // It is subdivided into static friction between non-moving surfaces and kinetic friction between moving surfaces.
+        // So used here is the dry friction coefficient of the paired materials (see reference).
+        // Dimensionless.
+        //
+        // Unit: []
+        //
+        // \par References:
+        // - http://adaptivemap.ma.psu.edu/websites/6_friction/dry_friction/dryfriction.html
+        //
+        optional double kinetic_friction_coefficient = 1;
+
+        // Contains the x, y and z-coordinate of the contact point of 
+        // the wheel, so that walking, torsion and deflation can be visualized.
+        // relative to the center of the wheel.
+        //
+        // Unit: [m]
+        //
+        optional Vector3d contact_point = 2;
+
+        // Contains the rotational speed of each wheel per second.
+        //
+        // Unit: [rad/s]
+        //
+        optional double rotational_speed = 3;
+
+        // Contains the steering angle of each wheel.
+        //
+        // Unit: [rad]
+        //
+        optional double steeringangle = 4;
+
+        // Contains the camber of each wheel.
+        // 
+        // Negative camber if the bottom of the wheel is farther out than the
+        // top.
+        // For more information: https://en.wikipedia.org/wiki/Camber_angle.
+        //
+        // Unit: [rad]
+        //
+        optional double camber = 5;
+
```

Comment:
Wheel **toe** for completness?

Reply:
at least it is the same like the steering-angle right?